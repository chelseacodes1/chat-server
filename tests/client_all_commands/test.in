CHANNELS
REGISTER user3 november3
REGISTER user3 november3
LOGIN user3
LOGIN user3 november30
LOGIN user3 november3
LOGIN user3 november3
CREATE local
JOIN local 
CHANNELS 
JOIN f2
JOIN f1
CREATE f1
JOIN f1
SAY f1 it's race weekend
SAY f1 in_Qatar
SAY local walking