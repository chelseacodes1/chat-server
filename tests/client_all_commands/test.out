RESULT CHANNELS f1, intros, news
RESULT REGISTER 1
RESULT REGISTER 0
RESULT LOGIN 0
RESULT LOGIN 0
RESULT LOGIN 1
RESULT LOGIN 0
RESULT CREATE local 1
RESULT JOIN local 1
RESULT CHANNELS f1, intros, local, news
RESULT JOIN f2 0
RESULT JOIN f1 1
RESULT CREATE f1 0
RESULT JOIN f1 0
RECV user3 f1 it's race weekend
RECV user3 f1 in_Qatar
RECV user3 local walking
